docs/src/overview.md

```diff
@@ -0,0 +1,102 @@
+
+6.	[Threaded Assembly](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/threaded_assembly/)  
+
+ This example shows the threaded assembly of a finite element stiffness matrix, i.e. the calculation is speed up by using parallelization. Different colorings of two-dimensional meshes are shown to visualize the split of the mesh in such a way that no threads interfere with each other.  
+	
+ Keywords: Parallelization, performance, threads, coloring
+
+7.	[von Mises Plasticity](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/plasticity/)  
+
+ A cantilever beam is solved applying a plasticity material model, requiring a constitutive driver. Handling of state, flux and internal variables within elements is shown. Newton’s method is used to iteratively solve the non-linear system.  
+	
+ Keywords: Plasticity, 3D, material modeling, material state, non-linear problem, Newton’s method, 
+
+8.	[Time Dependent Problems](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/transient_heat_equation/)
+  
+ The transient heat equation is solved on a rectangular plate. The time discretization is done by using the implicit Euler scheme.  
+	
+ Keywords: Time dependent problem, transient heat equation, implicit Euler scheme
+
+9.	[Ginzburg-Landau model energy minimization](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/landau/)  
+  
+ A basic Ginzburg-Landau model is solved by combining `Ferrite` with `ForwardDiff`. Using threads, the calculation time is optimized.  
+	
+ Keywords: Gizburg-Landau, ForwardDiff.jl, parallelization, optimization, performance
+
+10.	[Linear shell](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/linear_shell/)  
+
+ A program for the use of linear shell elements is set up. The theoretical background is introduced as well.  
+	
+ Keywords: Shell elements, displacements, rotations
+ 
+11.	[Nearly Incompressible Hyperelasticity](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/quasi_incompressible_hyperelasticity/)  
+
+ This program combines ideas from the examples Incompressible Elasticity and Hyperelasticity to construct a mixed element solving three-dimensional displacement-pressure equations.
+	
+ Keywords: Non-linear problem, hyperelasticity, finite strain, large deformations, Newton's method, automatic differentiation, coupled problem, mixed elements, displacement-pressure equation, incompressibility
+
+12.	[Incompressible Navier-Stokes Equations via DifferentialEquations.jl](https://ferrite-fem.github.io/Ferrite.jl/dev/examples/ns_vs_diffeq/)  
+
+ The weak form of the semi-discretized incompressible Navier-Stokes equations are derived from the strong form and implemented in the finite element code. Then, the time-dependent solution is calculated by using the DifferentialEquations.jl package. For this purpose, the PDE is required in a specific form.  
+	
+ Keywords: Fluid dynamics, weak and strong form, solver, incompressibility, time-dependent problem
```

Comment:
Can we move this into a separate list below the "pure" Ferrite examples? This is technically an example of how to integrate Ferrite.jl with DifferentialEquations.jl and qualitatively different from the other examples.

Reply:
Yes!

Reply:
changed in current version